inc/class-theme-assets.php

```diff
@@ -260,7 +260,7 @@ private function load_comments_assets() {
 				&& get_option( 'thread_comments' );
 
 			if ( $load_comments ) {
-				wp_enqueue_script( 'comment-reply' );
+				wp_enqueue_script( 'comment__reply' );
```

Comment:
OK please update all other areas where this is happening BEM applies only to HTML markup not PHP code.